exporter/datadogexporter/config/config.go

```diff
@@ -499,7 +516,7 @@ func (c *Config) Unmarshal(configMap *config.Map) error {
 	c.warnings = append(c.warnings, renamingWarnings...)
 
 	switch c.Metrics.HistConfig.Mode {
-	case histogramModeCounters, histogramModeNoBuckets, histogramModeDistributions:
+	case HistogramModeCounters, HistogramModeNoBuckets, HistogramModeDistributions:
```

Comment:
One last thing (sorry just saw this now): this switch can be removed since `UnmarshalText` will check for this

Reply:
@mx-psi 
Should I remove the following code?
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c049c656ca153cc5378add68e3351d38b7844e0e/exporter/datadogexporter/config/config.go#L518-L523

Reply:
Yes, that's it

Reply:
Removed it! https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/9589/commits/5eb122982d1cbfc09c17afa54a488051376199ef